core/src/main/res/layout/mz_view_map.xml

```diff
@@ -11,6 +11,20 @@
       android:layout_height="match_parent"
       />
 
+  <com.mapzen.android.graphics.CompassView
+      android:id="@+id/mz_compass"
+      android:layout_width="@dimen/mz_compass_button_width"
+      android:layout_height="@dimen/mz_compass_button_height"
+      android:layout_alignParentLeft="true"
```

Comment:
Ah yes, I didn't know that in GMaps it is on the right. But in Google Maps API it is on left side. I put it to the right because I needed to retain consistency with Google Maps API and not to confuse users. Also I have added zoom buttons like in Google Maps API and moved location button to the top right so this is why I put compass on left top. But yes, these are only my notes why I did this.